.changeset/poor-frogs-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': patch
+---
+
+enable access to cloudflare runtime [KV, R2, Durable Objects]
```

Comment:
Can you add a code example in the changeset of how this would be used? Also in the readme.